src/main/java/com/NewsWorkflw/common/controller/ModelAndView.java

```diff
@@ -0,0 +1,21 @@
+package com.NewsWorkflw.common.controller;
+

```

Comment:
Hmmmmm.  This looks like an empty class, which is clutter.  Clutter is tech debt.  If you create this class, integrate it with the program so that it has a purpose.